base/services/w32time/ntpclient.c

```diff
@@ -0,0 +1,183 @@
+/*
+ * PROJECT:     ReactOS Timedate Control Panel
+ * LICENSE:     GPL - See COPYING in the top level directory
+ * FILE:        dll/cpl/timedate/ntpclient.c
+ * PURPOSE:     Queries the NTP server
+ * COPYRIGHT:   Copyright 2006 Ged Murphy <gedmurphy@gmail.com>
+ *
+ */
```

Comment:
Wrong header type and wrong header contents

Reply:
This is an exact copy of the file in dll/cpl/timedate/ntpclient.c, so I left this alone. Can you suggest how it should be changed?

Reply:
It seems that this logic is supposed to be in w32time.dll only and exposed through the exported function W32TimeSyncNow. timedate.cpl can then call that function, which avoids the duplication.

Reply:
I used "depends" on w32time.dll from w2K3 and see the exported function W32TimeSyncNow, so I agree with your suggestion. Unfortunately, I have no idea how to turn my W32Time.exe into a DLL and add an exported function. Can you suggest helpful references for this or a direction for me to start? Thanks.

Reply:
Maybe we could plead this case to @gedmurphy and he could help here?

Reply:
I'm not sure what you're asking, can you clarify?

If you asking what you should do, then Thomas' suggestion is the way to go.
Are you asking for permission to move the file? In which case, of course, you don't need permission to make improvements.
Are you asking for my help to do the work Thomas suggested? I can certainly help with that.
Are you asking for me to do the work? Hmm, well, I could, but it might take a while for me to get around to it  ;)

Reply:
@gedmurphy, thank you for the kindness of your reply. For clarification, I suppose that it was a bit of all that you had mentioned above. It would be nice if you were to do the work :) but I know that you must be busy. Maybe I can see what I can do since it seems that you are OK with me including ntpclient.c into w32time.dll as an export. This is definitely a stretch for my capabilities. Thanks.